src/components/articleItems.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react"
+import { getAllArticles } from "./api"
+import { ArticleBox } from "./articleBox"
+
+export const ArticleItems = () => {
+    const [isLoading, setIsLoading] = useState(false);
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        setIsLoading(true);
+        getAllArticles().then((response) => {
+            setIsLoading(false);
+            if (response && response.articles) {
```

Comment:
This if condition isn't really necessary. If the promise rejects you'll get to a catch block with the error. In order to get to this if condition the promise has resolved. You wouldn't want to set the articles to an empty array as you already have that on line 7.